assets/scripts/segments/remove.js

```diff
@@ -24,7 +24,7 @@ export function removeSegment (el) {
   infoBubble.hideSegment()
 
   // Update the store
-  store.dispatch(removeSegmentActionCreator(window.parseInt(el.dataNo, 10)))
+  store.dispatch(removeSegmentActionCreator(window.parseInt(el.dataNo, 10), false))
```

Comment:
We may as well use this moment to replace `window.parseInt` with `Number.parseInt`